### ISSUE TITLE
[Suggestion] Graph of number of games vs game length

### ISSUE THREAD

**Issue by oribix** [joel1st/championweb#13]
Hi! I am a theory crafter that uses your website often to toss around ideas. I thought I would suggest it because I had an epiphany. While looking at some champions, I saw that some champions had very good winrates in the late game or in the early game, but their average winrate didn't reflect that they were reaching late game or rushing a win early.

I believe that it might be a good idea to have a graph for each champion that shows number of games vs game length. This way, people can say "Wow Vayne has such a high winrate lategame" but then look at the # of games vs game length and see that most of her games end before she even hits a 50% winrate. 

Basically, I think it would be good to add this feature so that we can compare this graph with the winrate vs game length graph.

I think the best way to showcase this data is to overlay the winrate vs game length with this new graph similar to how this graph from lolking looks:
http://i.imgur.com/dn04G5f.png

What do you think?
